### ISSUE TITLE
Generate a `solc_json` output

### ISSUE THREAD

**Issue by stas** [ApeWorX/ape-vyper#140]
### Overview

Trying to verify manually a deployed contract, but can't generate a `vyper -f solc_json` output.

Manually calling the `vyper` doesn't work because I have dependencies installed via `ape pm`.

### Specification

Ideally we get the `solc_json` output as a JSON file with the contract name somewhere in the `.build` folder.

**Comment by stas** [comment]
Current work-around is to manually call the `vyper` with the dependency paths. Eg.
```
/root/.vvm/vyper-0.4.0 -f solc_json -o lp1.json --path=/root/.ape/packages/projects/pcaversaccio_snekmate/0_1_0/src/ --evm-version cancun contracts/LpSugar.vy
```

Full context here:
https://github.com/velodrome-finance/sugar/pull/102/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557

**Comment by antazoey** [comment]
solc_json is not available yet in vyper when using `--standard-json`, I started adding it here: https://github.com/vyperlang/vyper/pull/4440

**Comment by stas** [comment]
@antazoey did you see my previous comment? I'm not sure where `--standard-json` is needed :thinking:

**Comment by antazoey** [comment]
> @antazoey did you see my previous comment? I'm not sure where `--standard-json` is needed 🤔

Yeah, sorry I was confused, I didn't realize this was a stand-alone output format.
We're working on a fix!

**Comment by stas** [comment]
Thanks so much!
